.github/workflows/jest-coverage-changes.yml

```diff
@@ -0,0 +1,26 @@
+name: Jest Coverage - changed files
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - release/v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - run: |
+          git checkout -b develop origin/develop
+          git checkout -
```

Comment:
why two git checkouts ?

Reply:
the actions/checkout GitHub Action does not actually checkout your full repository but only the code relevant to the PR. As such, `develop` as a branch did not exist.
we need both `current` and `develop` to run `jest --changedSince=develop`

second checkout here is to go back to current branch

refer: https://gist.github.com/bnb/9de89a07278e9f57cd058a535ab89a9b